doc/recipe_elements/software_selection.rst

```diff
@@ -23,3 +23,12 @@ Attributes:
 
 * ``id`` - environment id (as defined in comps file)
 * ``select`` (optional) - ``random`` (random environment selection)
+
+/installation/hub/software_selection/kernel_options
+===================================================
+Handles *Kernel Options* selection on the aarch64 architecture (RHEL-9 and higher).
+Currently, this option behaves correctly only during installations that have the
```

Comment:
If my suggestions WRT translation are correct and you find a way to use the translation properly, please remove this note.

Reply:
Thanks for the reminder!

Reply:
Removed!